packages/sit-onyx/src/components/OnyxForm/OnyxForm.core.ts

```diff
@@ -25,6 +26,13 @@ export type FormProps = {
    * See [:user-invalid](https://drafts.csswg.org/selectors/#user-invalid-pseudo).
    */
   showError?: ShowErrorMode;
+  /**
+   * Required mode: `optional` will show an `(optional)` text after the label for optional inputs.
```

Comment:
```suggestion
   * Required mode: `optional` will show an `(optional)` text after the label for optional form elements.
```

Its not limited to inputs, right? Also select, textarea etc,